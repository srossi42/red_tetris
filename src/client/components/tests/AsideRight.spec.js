import React from 'react'
import { shallow } from 'enzyme';
import AsideRight from '../asideRight'
import toJson from "enzyme-to-json"

describe('<AsideRight/> Component', () => {
  it('rendering correctly with no shapes', () => {
    let wrapper = shallow(
      <AsideRight scoreAdversary={null} boardAdversary={null} numberPlayer="1" name="lol"/>
    )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
