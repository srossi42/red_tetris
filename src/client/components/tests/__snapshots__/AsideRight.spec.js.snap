// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AsideRight/> Component rendering correctly with no shapes 1`] = `
<styled.div>
  <styled.div />
</styled.div>
`;
